Source/Widgets/Cesium3DTilesInspector/Cesium3DTilesInspectorViewModel.js

```diff
@@ -950,6 +1091,13 @@ define([
     };
 
     /**
+     * Toggles the visibility of the optimization section
+     */
+    Cesium3DTilesInspectorViewModel.prototype.toggleOptimization = function() {
+        this.optimizationVisible = !this.optimizationVisible;
+    };
```

Comment:
Put this block below `toggleTileDebugLabels` just so the visual arrangement matches the code arrangement.